cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -81,4 +81,12 @@
    * Option to a json encoded {@link ArtifactMeta} of the artifact containing the program.
    */
   public static final String ARTIFACT_META = "artifactMeta";
+
+  /**
+   * Options for impersonation
+   */
+  public static final String PRINCIPAL = "principal";
+
+  public static final String KEYTAB_PATH = "keytabPath";
```

Comment:
Is this always a local path or is the keytab URI in the namespace meta? BTW, it might be better to call the one in `NamespaceMeta` as `keytabURI` to reflect what it actually stores. Path usually refers to local path.